src/mediapipe_internal/mediapipegraphexecutor.cpp

```diff
@@ -101,7 +214,48 @@ static Status deserializeTensor(const std::string& requestedName, const KFSReque
     return StatusCode::OK;
 }
 
-static Status deserializeTensor(const std::string& requestedName, const KFSRequest& request, ov::Tensor& outTensor) {
+static Status deserializeTensor(const std::string& requestedName, const KFSRequest& request, TfLiteTensor** outTensor, const std::unique_ptr<tflite::Interpreter>& interpreter, size_t tfLiteTensorIdx) {
+    auto requestInputItr = request.inputs().begin();
+    auto status = getRequestInput(requestInputItr, requestedName, request);
+    if (!status.ok()) {
+        return status;
+    }
+    auto inputIndex = requestInputItr - request.inputs().begin();
+    auto& bufferLocation = request.raw_input_contents().at(inputIndex);
+    try {
+        auto datatype = getPrecisionAsTfLiteDataType(KFSPrecisionToOvmsPrecision(requestInputItr->datatype()));
+        std::vector<int> rawShape;
+        for (int i = 0; i < requestInputItr->shape().size(); i++) {
+            if (requestInputItr->shape()[i] < 0) {
+                std::stringstream ss;
+                ss << "Negative or zero dimension size is not acceptable: " << tensorShapeToString(requestInputItr->shape()) << "; input name: " << requestedName;
+                const std::string details = ss.str();
+                SPDLOG_DEBUG("[servable name: {} version: {}] Invalid shape - {}", request.model_name(), request.model_version(), details);
+                return Status(StatusCode::INVALID_SHAPE, details);
+            }
+            rawShape.emplace_back(requestInputItr->shape()[i]);
+        }
+        interpreter->SetTensorParametersReadWrite(
```

Comment:
Needs testing if passing nullptr as buffer is supported (0-data tensors)

Reply:
TfLite tensors in OVMS (de)serialization will not be supported in this release

Reply:
So why do we need this code?